specification/transport_layer/transport_layer.tex

```diff
@@ -534,6 +545,16 @@ \subsection{State variables}
     or the protocol stack can estimate this value automatically by analyzing the transfer intervals. \\
 \end{UAVCANSimpleTable}
 
+It might be useful to clarify why immediate switching between interfaces is not allowed,
```

Comment:
This is voiced conversationally whereas the rest of the specification is in "shall/should" voice. Could we adopt a box format to put non-normative and/or expositional text like this? It's jarring to be reading a spec and having a human suddenly take over the text and start talking.

Reply:
> It's jarring to be reading a spec and having a human suddenly take over the text and start talking.

A very good point. I suggest just keeping the style consistent throughout rather than adopting the box convention you suggested. Fixed this in my latest commit.